dist/src/main/config/broker.standalone.yaml.template

```
@@ -278,6 +278,10 @@
           # This setting can also be overridden using the environment variable ZEEBE_BROKER_DATA_BACKUP_S3_FORCEPATHSTYLEACCESS
           # forcePathStyleAccess: false
 
+          # When enabled, parts of the backup may be compressed.
```

```suggestion
          # When enabled, parts of the backup may be compressed. This helps to reduce the required storage space in s3 store. Enabling compression may use more cpu and disk while taking the backup.
```
Something along these line to indicate the pros/cons of enabling this would be useful.